tesseract4android/src/main/java/com/googlecode/leptonica/android/Skew.java

```diff
@@ -102,11 +103,20 @@ public static float findSkew(Pix pixs, float sweepRange, float sweepDelta, int s
 				sweepReduction, searchReduction, searchMinDelta);
 	}
 
+	public static Pix deskew(Pix pixs,int redSearch){
+		if (pixs == null)
+			throw new IllegalArgumentException("Source pix must be non-null");
+
+		return new Pix(nativeDeskew(pixs.getNativePix(),redSearch));
+	}
+
 	// ***************
 	// * NATIVE CODE *
 	// ***************
 
 	private static native float nativeFindSkew(long nativePix, float sweepRange, float sweepDelta,
 											   int sweepReduction, int searchReduction, float searchMinDelta);
 
+
```

Comment:
There should be only single empty line between the methods to make it consistent.